src/phoenix/server/api/types/Span.py

```diff
@@ -122,6 +124,14 @@ class Span:
         description="Cumulative (completion) token count from self and all "
         "descendant spans (children, grandchildren, etc.)",
     )
+    span_evaluations: List[SpanEvaluation] = strawberry.field(
+        description="Span evaluations",
```

Comment:
Nit, no need to repeat the name - best to be more verbose and informative in the descriptions.

Reply:
yea, that makes sense